decidim-core/lib/decidim/attributes/time_with_zone.rb

```diff
@@ -10,7 +10,10 @@ def coerce(value)
 
         Time.zone.strptime(value, I18n.t("time.formats.decidim_short"))
       rescue ArgumentError
-        nil
+        fallback = super
```

Comment:
Same thing here as above, I think the logic has to be slightly different with Virtus.

Reply:
Should be fixed by 8e35d4410e8871c6ca047b2776cfa3e23deb859d.